lib/github/tree.js

```diff
@@ -35,7 +35,15 @@ export default class GitHubTree {
       branch,
       path
     });
-    return data.repository.ref.target.history.nodes[0].oid;
+
+    const targetHistoryNodes = data.repository.ref.target.history.nodes;
+    if (!targetHistoryNodes.length) {
+      // Sometimes this function is in a spinner, and we need to print the error
+      // after a new line.
+      console.log();
+      throw new Error(`Cannot find WPT for "${path}". Please check the WPT name.`);
```

Comment:
Does `cli.stopSpinner(message, cli.SPINNER_STATUS.FAILED)` work here?

Reply:
Ah I see. It works much better.